src/points/PointsDiscoverCard.tsx

```diff
@@ -93,19 +101,24 @@ const styles = StyleSheet.create({
     ...typeScale.bodyXSmall,
     color: Colors.gray4,
   },
-  pill: {
+  balance: {
+    ...typeScale.labelSemiBoldSmall,
+    color: Colors.black,
+  },
+  linearGradient: {
+    height: 32,
```

Comment:
This appears to make it match the height of the previous pill.

```suggestion
    height: 40,
```

Reply:
confirmed with product to keep at 40, updated in [db3fe06](https://github.com/valora-inc/wallet/pull/6244/commits/db3fe06d0e5f4925c8a30ffd26f2c72fe2dcbbfc)